### PR TITLE
pulls.md: replace nodejs from the clone example

### PR DESCRIPTION
**Pull request by nikosalonen**
Fixes: https://github.com/TheBastionBot/BastionBotDevDocs/issues/1

I also added `git remote -v` to clarify what user should see.

### CODE REVIEW

File: docs/contributing/pulls.md [hunk 1]
```diff
@@ -22,12 +22,22 @@ and clone your fork locally and add the remote upstream branch.
...
-cd node
-git remote add upstream https://github.com/nodejs/node.git
+# Example: Bastion repository (https://github.com/TheBastionBot/Bastion.git)
+git clone https://github.com/YOUR_USERNAME/Bastion.git
+cd Bastion
+git remote add upstream https://github.com/TheBastionBot/Bastion.git
 git fetch upstream
 ```
 
+Verify the new upstream repository you've specified for your fork.
```
Comment (k3rn31p4nic) [inline docs/contributing/pulls.md]: According to me, *"Verify the upstream repository that you've specified for your fork is the original repository of Bastion"* is more correct. What do you think?

File: docs/contributing/pulls.md [hunk 2]
```diff
@@ -22,12 +22,22 @@ and clone your fork locally and add the remote upstream branch.
...
+```bash
+$ git remote -v
+origin    https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+origin    https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+upstream  https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (fetch)
```
Comment (k3rn31p4nic) [inline docs/contributing/pulls.md]: Directly mentioning the original user & repo name will be better I guess, so users can verify that they are doing it right.
```bash
 upstream  https://github.com/TheBastionBot/Bastion.git (fetch)
 upstream  https://github.com/TheBastionBot/Bastion.git (push)
```

Comment (nikosalonen) [reply docs/contributing/pulls.md]: Yeah that's correct since we've defined the example upstream

Comment (k3rn31p4nic) [reply docs/contributing/pulls.md]: If you could make the changes, I'd merge the PR right away. Thanks.

File: docs/contributing/pulls.md (hunk 1 above)
Comment (nikosalonen) [reply docs/contributing/pulls.md]: Yeah I'll change it.

**Comment by k3rn31p4nic** [comment]
Thanks for the contribution! You can now find yourself at https://bastionbot.org/contributors (as soon as GitHub API updates you as a contributor) and you can join the [Bastion HQ](https://discord.gg/fzx8fkt) and let me know your username and get your **Contributors** role.

Thanks!

**Pull request merged**